StaticAnalyzer/views/android/static_analyzer.py

```diff
@@ -189,10 +188,13 @@ def static_analyzer(request, api=False):
                     tracker = Trackers.Trackers(
                         app_dic['app_dir'], app_dic['tools_dir'])
                     tracker_res = tracker.get_trackers()
-                    dex_2_jar(app_dic['app_path'], app_dic[
-                              'app_dir'], app_dic['tools_dir'])
+
+                    if (settings.DECOMPILER == 'jadx'):
```

Comment:
we don't need this since we have only one decompiler by default. remove the if condition.